src/gettingstarted/local/docksal.md

```diff
@@ -0,0 +1,100 @@
+# Using Docksal for local development
+
+[Docksal](https://docksal.io) is a docker based local development tool that plays nicely with Platform.sh. It is maintained by a community of developers and is a viable option for most Platform.sh projects.
+
+See the [Docksal documentation](https://docs.docksal.io/) for installing and setting up Docksal on your system.
+
+Docksal will ask you to create a `.docksal` directory in your application root, which functions similarly to the `.platform.app.yaml` file. It is safe to check this directory into your Git repository as Platform.sh will simply ignore it.
+
+Docksal Fin (`fin`) is a command line tool for controlling Docksal and used for interacting with a Docksal project. For more information on the use of the `fin` command, type the `help` subcommand to get all available commands and options.
+
+```bash
+fin help
+```
+
+## Using Platform.sh CLI within Docksal
+
+The `SECRET_PLATFORMSH_CLI_TOKEN` must be set to use the Platform.sh CLI within your Docksal project. This is an API Token found with your Platform.sh account and can be generated by going to the [API Tokens](https://accounts.platform.sh/user/api-tokens) page and clicking the `Create API Token` link. This will allow for you to interact with your Platform.sh account from within the CLI container.
```

Comment:
Ah, I'd meant to link to the documentation page, but linking straight to the UI works too, I suppose.